libraries/standard/mqtt/include/mqtt.h

```diff
@@ -13,19 +34,18 @@ typedef struct MQTTContext MQTTContext_t;
 struct MQTTTransportInterface;
```

Comment:
```typedef struct MQTTContext MQTTContext_t;```

Does this need to be

```typedef struct MQTTContext * MQTTContext_t;```?

Reply:
No, we're not making these opaque.

Reply:
Oops not that one, I pointed out the wrong one